charts/argo-cd/values.yaml

```diff
@@ -532,3 +532,7 @@ configs:
     githubSecret: ""
     gitlabSecret: ""
     bitbucketSecret: ""
+#   argocdServerTlsConfig:
+#     key:
+#     cert:
+#   argocdServerAdminPassword:
```

Comment:
Would it be possible to include in what format you would expect the password to be as a comment?

As I understand it from <https://argoproj.github.io/argo-cd/faq/> Admin password reset section it would expect a bcrypt hash to be passed in here

Reply:
done